client/network.go

```diff
@@ -45,28 +40,16 @@ func (cli *NetworkCli) CmdNetwork(chain string, args ...string) error {
 // CmdNetworkCreate handles Network Create UI
 func (cli *NetworkCli) CmdNetworkCreate(chain string, args ...string) error {
 	cmd := cli.Subcmd(chain, "create", "NETWORK-NAME", "Creates a new network with a name specified by the user", false)
-	flDriver := cmd.String([]string{"d", "-driver"}, "bridge", "Driver to manage the Network")
-	cmd.Require(flag.Min, 1)
+	flDriver := cmd.String([]string{"d", "-driver"}, "", "Driver to manage the Network")
+	cmd.Require(flag.Exact, 1)
 	err := cmd.ParseFlags(args, true)
 	if err != nil {
 		return err
 	}
-	if *flDriver == "" {
-		*flDriver = defaultDriverType
-	}
 
 	// Construct network create request body
 	ops := make(map[string]interface{})
 	nc := networkCreate{Name: cmd.Arg(0), NetworkType: *flDriver, Options: ops}
-
```

Comment:
@aboch that's because, we have a concept of user-defined daemon default for network and driver. If specified as a daemon default (via Docker daemon flags or dnet toml or other means), then that must be used as a default. UI has no idea on any such defaults and hence it cannot make use of it. Hence it moves to the backend. 
Also, UI defaults are very different from backend implementation defaults (such as bridge is default bridge today). That is functionality dependent and the UI is just a reflection of that. So, moving to the backend is more appropriate. But the question of where in the backend is more appropriate. We have it at the API layer and not in libnetwork core. that is a design choice.